File: Dockerfile.cpp
# Sử dụng image gcc gốc của bạn
FROM gcc:latest

# Tải file json.hpp và lưu vào thư mục thư viện chuẩn của C++
RUN wget https://github.com/nlohmann/json/releases/download/v3.11.3/json.hpp -O /usr/include/json.hpp

# Terminal tại thư mục Backend và chạy lệnh này để đúc thành một Image mới (đặt tên là gcc-leetcode):
    # -> docker build -t gcc-leetcode -f Dockerfile.cpp .

File: Dockerfile.java
# Sử dụng image Java gốc của bạn
FROM eclipse-temurin:17-jdk-alpine

# Cài đặt wget (nếu Alpine chưa có) và tải file gson.jar về thư mục /opt/
RUN apk add --no-cache wget && \
    wget https://repo1.maven.org/maven2/com/google/code/gson/gson/2.10.1/gson-2.10.1.jar -O /opt/gson.jar

    # terminal -> docker build -t java-leetcode -f Dockerfile.java .
    